src/metabase/notification/task/send.clj

```diff
@@ -156,13 +167,13 @@
   ;; Get all existing triggers and subscription IDs
   (let [existing-triggers                  (:triggers (task/job-info send-notification-job-key))
         existing-triggers-subscription-ids (map #(get-in % [:data "subscription-id"]) existing-triggers)
-        subscription-id->cron              (t2/select-pk->fn identity :model/NotificationSubscription :type :notification-subscription/cron)
-        db-subscription-ids                (keys subscription-id->cron)
-        [to-delete to-create _to-skip]     (diff existing-triggers-subscription-ids db-subscription-ids)]
+        subscription-id->subscription      (active-cron-subscription-id->subscription)
+        db-subscription-ids                (keys subscription-id->subscription)
+        [to-delete to-create _to-skip]     (diff (set existing-triggers-subscription-ids) (set db-subscription-ids))]
```

Comment:
turning these to set will fix the exception on stats.

apparently:
```clojure
(diff (keys {1 :a
             2 :b})
      (lazy-seq [1]))
;; => [[nil 2] nil [1]]
```

Reply:
wtf; maybe it's better to handle that inside of the `diff` it's ours?

Reply:
i'm not sure I understand that. are you suggesting we use diff utils from our namespaces?

Reply:
ahh it's `clojure.data/diff`. Function name w/o namespace tricked me into thinking it's a local one.